### PR TITLE
add expand/collapse buttons to doc preview popup

### LINKED ISSUE #182: Add a button to expand the document preview full-screen
This might be something provided by lightbox and not straightforward to add—investigation will be required.

### PR DESCRIPTION
**Pull request by janebitovi**
fixes #182 - add expand/collapse buttons to doc preview popup

### CODE REVIEW

File: src/modal/modal.stache [hunk 1]
```diff
@@ -6,15 +6,21 @@
...
     aria-labelledby="pageModalLbl"
   >
 
     <div class="modal-dialog {{#if(modalContent.textlongFieldVM.field.name)}}modal-lg{{/if}}">
       <div class="modal-content">
         {{#if(modalContent)}}
         <div class="modal-header">
-          <button type="button" class="close" data-dismiss="modal" aria-label="Close" aui-action="close" tabindex="1">
+          {{#if(allowFullscreen)}}
+            <button type="button" class="fullscreen" on:click="toggleFullscreen()" aria-label="Fullscreen" tabindex="1">
```
Comment (chasenlehara) [inline src/modal/modal.stache]: Does the `tabindex` need to be on the button? I think it should be focusable without it.

I would consider adding a `title` attribute to this too, so mouse users can hover over it and see what it does.

Comment (janebitovi) [reply src/modal/modal.stache]: The tab index is already set to 1 elsewhere in the modal so adding it to the new button keeps everything tabbing how you'd expect. Didn't want to change legacy stuff too much for a last min addition.

title attribute makes sense, a11y audit probably would want it on the close button too, yeah? I'll add it to both

Comment (chasenlehara) [reply src/modal/modal.stache]: Yeah, that’d be helpful.

I don’t think we should have the `tabindex`, but I’d say leave it for consistency and if I discover problems with it, I’ll submit another PR.

**Comment by chasenlehara** [comment]
This looks great! One issue at larger screen sizes though—the preview seems to cut off at the bottom:

A document preview with a lot of text causes the bottom to be cut off

**Comment by janebitovi** [comment]
addressed each
image
image
image

fullscreen2

**Pull request merged**